plugin_runner/tests/fixtures/plugins/test_load_questionnaire/protocols/my_protocol.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+from canvas_sdk.effects import Effect
+from canvas_sdk.events import EventType
+from canvas_sdk.protocols import BaseProtocol
```

Comment:
I would like to start moving away from using `BaseProtocol` as a default, and instead using `BaseHandler`. They aren't different at the moment but they will be at some point, and in most cases `BaseHandler` will be more appropriate.

Reply:
Happy to approve once these get changed to inheriting from `BaseHandler` instead of from `BaseProtocol`